src/Controls/tests/UITests/Tests/Issues/Issue18613.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18613 : _IssuesUITest
+	{
+		public Issue18613(TestDevice device) : base(device) { }
+
+		public override string Issue => "Disabled Button cannot trigger Command";
+
+		[Test]
+		public void CommandDisableButton()
+		{
+			App.WaitForElement("WaitForStubControl");
+
+			// Appium cannot find a disable Button using FindElement
+			if (Device != TestDevice.iOS && Device != TestDevice.Mac)
+			{
+				var button = App.FindElement("ButtonControl");
+
+				if (button is not null)
+				{
+					button.Click();
+
+					// If clicking the Button we do not have a counter result in
+					// the Label, the command has not been executed.
+					var counter = App.FindElement("CounterLabel").GetText();
+					Assert.True(string.IsNullOrEmpty(counter));
+				}
+			}
+
+			// Verify the visual status of the disabled Button.
+			VerifyScreenshot();
```

Comment:
I don't see a macos screenshot? This should still run?

Reply:
MacCatalyst isn't supported yet for visual test.